packages/frame-core/src/schemas/shared.ts

```diff
@@ -0,0 +1,23 @@
+import { z } from 'zod';
+
+export const secureUrlSchema = z.string().url().startsWith('https://', { message: 'Must be an https url' }).max(512);
+
+export const frameNameSchema = z.string().max(32);
+
+export const colorValueSchema = z.string().max(9).optional();
```

Comment:
do we want to have it start with `#` to prevent people using color names which may not always work?

Reply:
added a hex regex